script/main.py

```diff
@@ -142,42 +142,42 @@ def insert_users_to_table(conn, users):
 
 
 def insert_users_to_db(conn, people):
-  people = []
+  users = []
 
   list_of_people = people['results']
-  for dict_of_person in list_of_people:   
-    gender = dict_of_person['gender']
-    name_title = dict_of_person['name']['title']
-    name_first = dict_of_person['name']['first']
-    name_last = dict_of_person['name']['last']
-    location_street_number = dict_of_person['location']['street']['number']
-    location_street_name = dict_of_person['location']['street']['name']
-    location_city = dict_of_person['location']['city']
-    location_state = dict_of_person['location']['state']
-    location_country = dict_of_person['location']['country']
-    location_postcode = dict_of_person['location']['postcode']
-    location_coordinates_latitude = dict_of_person['location']['coordinates']['latitude']
-    location_coordinates_longitude = dict_of_person['location']['coordinates']['longitude']
-    location_timezone_offset = dict_of_person['location']['timezone']['offset']
-    location_timezone_description = dict_of_person['location']['timezone']['description']
-    email = dict_of_person['email']
-    login_uuid = dict_of_person['login']['uuid']
-    login_username = dict_of_person['login']['username']
-    login_password = dict_of_person['login']['password']
-    login_salt = dict_of_person['login']['salt']
-    login_md5 = dict_of_person['login']['md5']
-    login_sha1 = dict_of_person['login']['sha1']
-    login_sha256 = dict_of_person['login']['sha256']
-    dob_date = dict_of_person['dob']['date']
-    dob_age = dict_of_person['dob']['age']
-    dob_time_until_birthday = dict_of_person['dob']['time_until_birthday']
-    registered_date = dict_of_person['registered']['date']
-    registered_age = dict_of_person['registered']['age']
-    phone = dict_of_person['phone']
-    cell = dict_of_person['cell']
-    id_name = dict_of_person['id']['name']
-    id_value = dict_of_person['id']['value']
-    nat = dict_of_person['nat']
+  for dict_of_people in list_of_people:   
```

Comment:
Usually you don't add data structure name to the variable name:
So instead of `list_of_peoples` it is better to name it just `peoples`
`dict_of_people` => `person`